### PR TITLE
feat: qr code in presentation mode

### PR DESCRIPTION
**Pull request by admbtlr**
### Description

This PR resolves [MOPLAT-445]

image

### PR Checklist

- [x] I tested my changes on **iOS** and **Android**.
- [x] I added screenshots or videos to illustrate my changes.
- [ ] I added Tests and Stories for my changes.

### To the reviewers 👀

- [ ] I would like **at least one** of the reviewers to run this PR on the simulator or device.

Need help with something? Have a look at our [docs], or get in touch with us.

[docs]: /docs/README.md

[MOPLAT-445]: https://artsyproduct.atlassian.net/browse/MOPLAT-445?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/app/sharedUI/screens/Artwork/ArtworkContent/ArtworkContent.tsx [hunk 1]
```diff
@@ -183,7 +198,7 @@ export const ArtworkContent = ({ slug }: { slug: string }) => {
           )}
           {!!additionalInformation && (
             <>
-              <Markdown>{additionalInformation}</Markdown>
+              <Text variant="xs"><Markdown>{additionalInformation}</Markdown></Text>
```
Comment (pvinis) [inline src/app/sharedUI/screens/Artwork/ArtworkContent/ArtworkContent.tsx:201]: is this extra text actually doing anything? 🤔 
i think all the styling within this markdown is based on the rules within the `Markdown` component, `defaultRules`.

**Review by pvinis — COMMENTED** [review]
nice. i only had one other thought:
obviously non-blocking, but how about we slap our logo in the middle of that QR code? 🤩

**Review by pvinis — APPROVED** [review]

File: src/app/sharedUI/screens/Artwork/ArtworkContent/ArtworkContent.tsx (hunk 1 above)
Comment (admbtlr) [reply src/app/sharedUI/screens/Artwork/ArtworkContent/ArtworkContent.tsx:201]: Oh whoops sorry this leaked in from something else I was playing around with. I'll take it out.

File: package.json [hunk 2]
```diff
@@ -62,6 +62,7 @@
     "react-native-haptic-feedback": "^1.13.0",
     "react-native-linear-gradient": "^2.5.6",
     "react-native-pager-view": "^5.4.25",
+    "react-native-qrcode-generator": "^1.2.2",
```
[the remark this answers is not in the document]
Comment (admbtlr) [reply package.json:65]: Yeah, but that one requires react-native-svg, which is a pretty big library, so I thought I'd use this one, which uses a `canvas` in a webview. Open to using the other library if you think that's not a problem though.

**Comment by admbtlr** [comment]
> nice. i only had one other thought: obviously non-blocking, but how about we slap our logo in the middle of that QR code? 🤩

ha! No possible with this library, I don't think

File: package.json (hunk 2 above)
[the remark this answers is not in the document]
Comment (pvinis) [reply package.json:65]: ah gotcha. ok makes sense, thanks for explaining.

**Comment by pvinis** [comment]
if you wanna try, we could totally just overlay the artsy logo on top, in the middle ;), that's what the QR redundancy data is for :D

File: src/app/sharedUI/screens/Artwork/ArtworkContent/ArtworkContent.tsx (hunk 1 above)
Comment (admbtlr) [inline src/app/sharedUI/screens/Artwork/ArtworkContent/ArtworkContent.tsx:201]:
```suggestion
              <Markdown>{additionalInformation}</Markdown>
```

**Comment by patrinoua** [comment]
I'll go ahead and make a PR to revert these changes as it is breaking the tests. Context here:
https://artsy.slack.com/archives/C02NAJ2CGLW/p1664965136471759

**Pull request merged**